src/logger/constants.ts

```diff
@@ -0,0 +1,14 @@
+import pino from "pino"
+
+// eslint-disable-next-line import/prefer-default-export
+export const PINO_OPTIONS = {
+  // NOTE: DO NOT format time in-process
+  // as this will affect perf of the logger.
+  // See here: https://getpino.io/#/docs/api?id=timestamp-boolean-function
+  timestamp: pino.stdTimeFunctions.isoTime,
+  errorKey: "error",
```

Comment:
we are not yet injecting the error / message in json for our log statements right?

Reply:
we are!